sdks/python/apache_beam/coders/coders.py

```diff
@@ -450,6 +451,40 @@ def value_coder(self):
     return self
 
 
+class ProtoCoder(FastCoder):
+  """A Coder for Google Protocol Buffers.
+
+  It supports both Protocol Buffers syntax versions 2 and 3. However,
+  the runtime version of the python protobuf library must exactly match the
+  version of the protoc compiler what was used to generate the protobuf
+  messages.
+
+  ProtoCoder is registered in the global CoderRegistry as the default coder for
+  any protobuf Message object.
+
+  """
+
+  def __init__(self, proto_message_type):
+    self.proto_message_type = proto_message_type
```

Comment:
The generated proto message hard codes the python module to the directory structure of where the proto file lives. I was placing the generated file in a different directory, and hence the module mismatch. Added instructions on how to generate it correctly in the comments of the generated file.